src/core/utils.js

```diff
@@ -286,6 +286,25 @@ export function normalizePadding(text = "") {
   return result;
 }
 
+/**
+ * Removes common indents across the IDL texts,
+ * so that indentation inside <pre> won't affect the rendered result.
+ * @param {string} text IDL text
+ */
+export function unindentMarkup(text) {
```

Comment:
`unindentMarkup` as name doesn't seem very right. move to `reindentMarkup`?

Reply:
IMO we don't need `Markup` to make it general, so maybe just `reindent`, `reduceIndent` or anything proper.

I used `unindent` because it basically does `Shift+Tab` 😁

Reply:
`rebaseIndent` :laughing: